variables.tf

```diff
@@ -122,8 +128,8 @@ variable "logging_bucket_prefix" {
   default     = ""
 }
 
-variable "logging_bucket_retention" {
-  description = "The number of days to retain load balancer logs. 0 to ratain forever."
+variable "s3_flowlog_retention" {
```

Comment:
We should drop this down farther into the file, somewhere around line 221, between `public_subnets_per_az` and `spoke_vpc`

Reply:
you right, I missed it .